js/levels.js

```diff
@@ -16,7 +16,7 @@ function createFirstLevel()
     [door.newlines, door.orchestra, door.Forbidden, door.chocolate, door.guitarCase, door.winter2],
     [door.sea, door.sofa, door.outdoor, door.chessMate, door.texture, door.chessStale],
     [door.drawnbyme, Superhero.art, PlayerStartsAt(door.black), door.green, door.banner, door.threeHeads],
-    [door.staircase, door.highLow, door.new, door.wheel, door.plain, door.top],
+    [door.quirky,door.staircase, door.highLow, door.new, door.wheel, door.plain, door.top],
```

Comment:
Please remove a tile and add another, or else a tile can't be entered.

Reply:
alright so I remove door.both from the second last line. Is it fine now?